lib/dor/services/client/mutate.rb

```diff
@@ -33,7 +33,10 @@ def apply_admin_policy_defaults
         # @raise [BadRequestError] when ETag not provided.
         # @return [Cocina::Models::DROWithMetadata,Cocina::Models::CollectionWithMetadata,Cocina::Models::AdminPolicyWithMetadata] the returned model
         # rubocop:disable Metrics/AbcSize
+        # rubocop:disable Metrics/MethodLength
         def update(params:, skip_lock: false)
+          raise ArgumentError, 'Cocina object not provided.' unless params.respond_to?(:externalIdentifier)
```

Comment:
hmm, i just merged, but after glancing back at this again, i'm wondering if `cocina_object` or similar might be a more informative parameter name that `params`

Reply:
It entirely would, but I don't want to change the method signature at this time.

Reply:
totally fair, and tbh i wasn't thinking about the caller's use of named args when i made that observation.